### PR TITLE
Feat/component warning

### PR DESCRIPTION
**Pull request by Rafael-Yokoyama**

### CODE REVIEW

File: src/components/Warning/index.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+interface IWarning {
+  title: string;
+  paragraph: string;
+}
+
+export function Warning({ title, paragraph }: IWarning) {
+  return (
+    <Container>
+      <h2> {title}</h2>
```
Comment (rodolfomori-devclub) [inline src/components/Warning/index.tsx:11]: keep the spacing pattern.

File: src/styles/themes/default.tsx [hunk 2]
```diff
@@ -6,4 +6,6 @@ export const defaultTheme = {
     lightBackground: "#FFFFFF",
     textColor: "#494949",
     lightTextColor: "#ADADAD",
+    warning:"#f95632",
+    BgWarning:"#ffe2e2"
```
Comment (rodolfomori-devclub) [inline src/styles/themes/default.tsx:10]: would it be better to use bgWarning ?

Comment (Rafael-Yokoyama) [reply src/styles/themes/default.tsx]: Yes , I agree

File: src/components/Warning/index.tsx (hunk 1 above)
Comment (Rafael-Yokoyama) [reply src/components/Warning/index.tsx]: Done

**Pull request merged**